cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java

```diff
@@ -48,6 +49,9 @@
     private AtomicReference<String> errorMessage = new AtomicReference<>("");
     private boolean isDebugMode = false;
 
+    @Autowired
```

Comment:
Cannot autowire here
See io.cloudslang.lang.cli.services.ScoreServicesImpl#triggerSync where we have new SyncTriggerEventListener()
We need to have a static method to print in the ConsolePrinter to access it from here with java method call.